C/CUDA/CUDA_Driver/init.jl

```diff
@@ -28,82 +28,160 @@ end
 
 libcuda_deps = [libcuda_debugger, libnvidia_nvvm, libnvidia_ptxjitcompiler]
 libcuda_system = Sys.iswindows() ? "nvcuda" : "libcuda.so.1"
-can_use_compat = true
+
+# if anything goes wrong, we'll use the system driver
+global libcuda = libcuda_system
 
 # check if we even have an artifact
 if @isdefined(libcuda_compat)
     @debug "Forward-compatible driver found at $libcuda_compat"
 else
     @debug "No forward-compatible driver available for your platform."
-    can_use_compat = false
+    return
```

Comment:
I was momentarily confused by this return, but we interpolate this file into a function right?

Reply:
Yeah. Let's hope nothing gets added after the `__init__` that JLLWrappers generates :-)